packages/vuetify/src/services/icons/presets/index.ts

```diff
@@ -1,11 +1,16 @@
 // Types
 import { VuetifyIconSets } from 'vuetify/types/services/icons'
 
+import mdiSvg from './mdi-svg'
 import md from './md'
 import mdi from './mdi'
 import fa from './fa'
 import fa4 from './fa4'
 
 export default Object.freeze({
-  md, mdi, fa, fa4
+  'mdi-svg': mdiSvg,
```

Comment:
There is no relation between that and mdi. It works with any svg path. Imho it should be `svg`.